core/app/services/spree/create_order.rb

```diff
@@ -0,0 +1,16 @@
+module Spree
+  class CreateOrder
+    def call(user:, store:, order_params: nil)
+      order_params ||= {}
+
+      default_params = {
+        user: user,
+        store: store,
+        currency: Spree::Config[:currency],
```

Comment:
@bbonislawski we should have more flexibility here - currency should also be a param